material-overrides/assets/stylesheets/polkadot.css

```diff
@@ -1371,16 +1374,31 @@ details.interface > summary::before {
 
 /** Feedback widget styling **/
 .md-feedback {
+  display: flex;
+  justify-content: center;
+  align-items: center;
   width: fit-content;
-  padding: 1em;
-  border: 1px solid var(--mystic);
-  border-radius: 16px;
-  margin: 1rem auto;
+  padding: 0;
+  margin-top: 0px;
+  margin-bottom: 0px;
 }
 
 .md-feedback__title {
   font-weight: 400;
-  font-size: 14px;
+  text-align: center;
+  white-space: nowrap;
+  font-size: 12px;
+  margin: 0;
+}
+
+.md-feedback__list{
```

Comment:
Can you please run prettier on this? It will add the space between the class and the parenthesis and clean up anything else that might need it

Reply:
Also, looks like there may be some unnecessary styles here - please check